playground/AzureContainerApps/AzureContainerApps.AppHost/postgres.module.bicep

```diff
@@ -0,0 +1,63 @@
+@description('The location for the resource(s) to be deployed.')
```

Comment:
Where's this file come from?

Reply:
Not my codez

Reply:
You added the file, and this playground app. I don't see anything in the app using postgres. I think this is a left over file that should be deleted.

Reply:
Oh I might have undone that change.